src/Controls/tests/DeviceTests/Elements/Label/LabelTests.cs

```diff
@@ -410,6 +411,84 @@ public async Task FontStuffAppliesEvenInHtmlMode()
+			};
+
+			var labelCenter = new Label
+			{
+				HorizontalOptions = LayoutOptions.Center,
+				LineBreakMode = lineBreakMode,
+				HorizontalTextAlignment = textAlignment,
+				Text = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+			};
+
+			var labelEnd = new Label
+			{
+				HorizontalOptions = LayoutOptions.End,
+				LineBreakMode = lineBreakMode,
+				HorizontalTextAlignment = textAlignment,
+				Text = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+			};
+
+			var labelFill = new Label
+			{
+				HorizontalOptions = LayoutOptions.Fill,
+				LineBreakMode = lineBreakMode,
+				HorizontalTextAlignment = textAlignment,
+				Text = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+			};
+
+			var layout = new VerticalStackLayout()
+				{
+					labelStart,
+					labelCenter,
+					labelEnd,
+					labelFill,
+				};
+
+			layout.HeightRequest = 300;
+			layout.WidthRequest = 100;
+
+			await InvokeOnMainThreadAsync(async () =>
+			{
+				var contentViewHandler = CreateHandler<LayoutHandler>(layout);
+				await contentViewHandler.PlatformView.AttachAndRun(() =>
+				{
+					Assert.Equal(double.Round(labelStart.Width, 5), double.Round(layout.Width, 5));
```

Comment:
I may be misunderstanding, but this line is just comparing the labelStart.Width to the layout.Width and rounding both of those numbers to the 5th decimal place. This was just added since the Android devices rounded the numbers to different place values

Reply:
Oh or are you suggesting that I need to set a label font size?

Reply:
Ah, I misunderstood what the test was doing. Ignore this one, sorry. :)